r.css

```
@@ -0,0 +1,240 @@
+        
+*{
+    padding: 0;
+    margin: 0;
+    
+}
+a{
+    text-decoration: none;
+    color:black;
+}
+header a:hover{
+    color:white;
+    transition-duration: 0.5s;
+    transition-delay: 0.5s;
+    font-size: 50px;
+    
+}
+
+html{
+  
+    background: linear-gradient(45deg,rgb(240, 236, 236),rgb(148, 143, 143));
+
+    height: 100%;
+}
+
+body{
+    height: 100%;
+    font-family: "Segoe UI", "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, sans-serif;
+
+}
+
+.wrapper{
+    height: 100%;
+}
+
+.grid_container{
+    height: 100%;
+    display: grid;
+    grid-template-columns: 100%;
+    grid-template-rows: 100px auto 210px;
+}
+
+.grid_container header{
+    display: grid;
+    align-content: center;
+    font-size: 40px;
+    background-color: rgb(193, 226, 155);
+    padding-left:40px;
+    font-family: Tahoma, sans-serif;
+    background: linear-gradient(45deg,rgb(121, 199, 121),rgb(187, 185, 185));
+    
+
+}
+
+.grid_container footer{
+    
+    background: linear-gradient(40deg,rgb(34, 40, 46),rgba(40, 47, 56, 0.637));
+    border-top: 1px rgba(95, 93, 93, 0.281) inset;
+    display: grid;
+    grid-template-columns: 3fr 4fr;
+    grid-template-rows: 10fr 5fr;
+    color:white;
+    
+   
+    
+}
+.grid_container footer .contacts_container{
+    display: grid;
+    grid-template-columns: 1fr ;
+    grid-template-rows: 1fr 2fr;
+    padding:15px 80px;
+    padding-left: 40%;
+    padding-bottom: 0;
+    
+}
+
+.grid_container footer .contacts_container .bar {
+    display: grid;
+   grid-template-columns: 15% 15% 15% auto;
+   grid-column-gap: 10%;
+   grid-auto-rows: minmax(5%,45%);
+   grid-auto-columns: minmax(5%,15%);
+ 
+}
+.grid_container footer .contacts_container .bar a{
```

Need space everywhere before `{`
```suggestion
.grid_container footer .contacts_container .bar a {
```